homeassistant/components/nasweb/coordinator.py

```diff
@@ -98,6 +105,8 @@ def __init__(
         self._listeners: dict[CALLBACK_TYPE, tuple[CALLBACK_TYPE, object | None]] = {}
         data: dict[str, Any] = {}
         data[KEY_OUTPUTS] = self.webio_api.outputs
+        data[KEY_INPUTS] = self.webio_api.inputs
+        data[KEY_TEMP_SENSOR] = self.webio_api.temp_sensor
```

Comment:
It's a little bit odd to use constants from the library to index data which we construct and consume ourselves